src/components/utils/rhf-inputs/ag-grid-table-rhf/bottom-right-buttons.js

```diff
@@ -0,0 +1,96 @@
+/**
```

Comment:
we have a similar component used with dndTable : DndTableBottomRightButtons. can it be used here ?

Reply:
yes, done